File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        index = -1
        ans = ''
        for i in range(min([len(s) for s in strs])):
            letter = strs[0][i]
            for string in strs:
                if string[i] != letter:
                    return ans
            
            ans += letter
        return ans
            

File: 0014-longest-common-prefix/0014-longest-common-prefix.ts
function longestCommonPrefix(strs: string[]): string {
    let res: string = '';
    const minLength: number = Math.min(...strs.map((val) => val.length));
    for (let i: number = 0; i < minLength; i++) {
        const letter: string = strs[0][i];
        for (const str of strs) {
            if (str[i] !== letter) {
                return res;
            };
        };
        res += letter;
    };

    return res;
};